tripgo.swagger.yaml

```diff
@@ -89,6 +89,223 @@ paths:
+              operator:
+                type: string
+                description: TSP name from `operatorInfo.json`.
+              modes:
+                type: array
+                items:
+                  type: string
+                description: Public transit modes (pt_pub_*) for which results should be returned. [Mode identifier](http://skedgo.github.io/tripgo-api/site/advanced/#mode-identifiers) strings like `pt_pub` or `pt_pub_subway`. In absence, all modes are included.
+              route:
+                type: string
+                description: Route name to retrieve information from. When ´full´ parameter is false, this field is ignored.
+              onlyRealTime:
+                type: boolean
+                description: Boolean to filter only routes with Real Time support. Only valid when ´full´ parameter is false. When true, returned route list will contain only routes with Real Time service.
+              full:
+                type: boolean
+                description: Boolean if result should include full information for route/s or just a list with all their names ignoring ´mode´ array in response. False value by default.
+            required:
+              - region
+              - operator
+            example:
+              - region: US_CA_LosAngeles
+                operator: Metro - Los Angeles
+                modes:
+                  - pt_pub_tram
+                  - pt_pub_bus
+                full: true
+      responses:
+        200:
+          description: Successful response
+          schema:
+            type: array
+            items:
+              type: object
+              description: Route information.
+              properties:
+                name:
+                  type: string
+                  description: Route name.
+                mode:
+                  type: array
+                  items:
+                    description: Detailed information for this route under a specific public transport mode. It could be the case that same route name is used by this operator under another mode.
+                    type: object
+                    properties:
+                      numberOfServices:
+                        type: integer
+                        description: Total number of services for this route and mode.
+                      stops:
+                        type: array
+                        items:
+                          type: string
+                          description: Stop code. DO WE USE STOP CODES TO REFER STOPS IN OUR ENDPOINTS?
+                        description: Used stops set for this route.
+                      realTime:
+                        type: object
+                        $ref: '#/definitions/RealtimeSnapshot'
+            example:
+              code: ONCE WE HAVE RESPONSE MODEL DEFINED
+        400:
+          description: SHOULD I PUT ERROR MODES FOR SPECIFIC SCENARIOS LIKE NON-VALID FILTER USED, ETC?
+
+  /tripInfo.json:
```

Comment:
We shouldn't use the term "trip" here. Trips in the lingo of our API are a-to-b trips produced by the routing endpoints.

I think we should call these "service". Their IDs can be used as input to `service.json`, correct?